content/blog/2013-06-02-jsfiddle-integration.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: JSFiddle Integration
+title: Integración con JSFiddle
 author: [vjeux]
 ---
 
-[JSFiddle](https://jsfiddle.net) just announced support for React. This is an exciting news as it makes collaboration on snippets of code a lot easier. You can play around this **[base React JSFiddle](http://jsfiddle.net/vjeux/kb3gN/)**, fork it and share it! A [fiddle without JSX](http://jsfiddle.net/vjeux/VkebS/) is also available.
 
+[JSFiddle](https://jsfiddle.net) acaba de anunciar soporte para React. Esta es una noticia increible debido a que hace la colaboración con fragmentos de código mucho más fácil. ¡Puedes jugar, bifurcar y compartir esta **[base para react en JSFiddle](http://jsfiddle.net/vjeux/kb3gN/)**! Un [fiddle sin JSX](http://jsfiddle.net/vjeux/VkebS/) también esta disponible.
```

Comment:
* `increíble`.
* I suggest `hace la colaboración de fragmentos de código`.
* I'd put `forkear` instead of `bifurcar` to make this clearer.
* también `está` disponible.

Reply:
done :)